docs/tutorials/qaoa_with_sampler.ipynb

```diff
@@ -0,0 +1,338 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import Pauli, SparsePauliOp\n",
+    "\n",
+    "\n",
+    "def get_operator(weight_matrix: np.ndarray) -> tuple[SparsePauliOp, float]:\n",
+    "    r\"\"\"Generate Hamiltonian for the graph partitioning\n",
+    "    Notes:\n",
+    "        Goals:\n",
+    "            1 Separate the vertices into two set of the same size.\n",
+    "            2 Make sure the number of edges between the two set is minimized.\n",
+    "        Hamiltonian:\n",
+    "            H = H_A + H_B\n",
+    "            H_A = sum\\_{(i,j)\\in E}{(1-ZiZj)/2}\n",
+    "            H_B = (sum_{i}{Zi})^2 = sum_{i}{Zi^2}+sum_{i!=j}{ZiZj}\n",
+    "            H_A is for achieving goal 2 and H_B is for achieving goal 1.\n",
+    "    Args:\n",
+    "        weight_matrix: Adjacency matrix.\n",
+    "    Returns:\n",
+    "        Operator for the Hamiltonian\n",
+    "        A constant shift for the obj function.\n",
+    "    \"\"\"\n",
+    "    num_nodes = len(weight_matrix)\n",
+    "    pauli_list = []\n",
+    "    coeffs = []\n",
+    "    shift = 0\n",
+    "\n",
+    "    for i in range(num_nodes):\n",
+    "        for j in range(i):\n",
+    "            if weight_matrix[i, j] != 0:\n",
+    "                x_p = np.zeros(num_nodes, dtype=bool)\n",
+    "                z_p = np.zeros(num_nodes, dtype=bool)\n",
+    "                z_p[i] = True\n",
+    "                z_p[j] = True\n",
+    "                pauli_list.append(Pauli((z_p, x_p)))\n",
+    "                coeffs.append(-0.5)\n",
+    "                shift += 0.5\n",
+    "\n",
+    "    for i in range(num_nodes):\n",
+    "        for j in range(num_nodes):\n",
+    "            if i != j:\n",
+    "                x_p = np.zeros(num_nodes, dtype=bool)\n",
+    "                z_p = np.zeros(num_nodes, dtype=bool)\n",
+    "                z_p[i] = True\n",
+    "                z_p[j] = True\n",
+    "                pauli_list.append(Pauli((z_p, x_p)))\n",
+    "                coeffs.append(1.0)\n",
+    "            else:\n",
+    "                shift += 1\n",
+    "\n",
+    "    return SparsePauliOp(pauli_list, coeffs=coeffs), shift\n",
+    "\n",
+    "\n",
+    "qubit_op, offset = get_operator(w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize service and select a backend\n",
+    "\n",
+    "Next, we will create a service instance and specify a backend. In this example we will use a simulator to avoid queue times, but you can use the following code to run on a real device by simply changing the backend."
```

Comment:
Do we need to mention access credentials here?

Reply:
Good point, I added a line about that in [7b3af55](https://github.com/Qiskit/qiskit-ibm-runtime/pull/888/commits/7b3af558903cb669db54ebd60be1d75f6dfe73bd)